src/test/data/JsonTrackrStorageTest/invalidPersonTrackr.json

```diff
@@ -4,5 +4,10 @@
     "phone": "9482424",
     "email": "hans@example.com",
     "address": "4th street"
+  } ],
+  "tasks" : [ {
```

Comment:
Since the test case is focused on testing invalid person, can we just leave the tasks array empty? (Same for the other tests that are solely testing methods related to person")